lib/nylas/v2.rb

```diff
@@ -1,6 +0,0 @@
-require_relative 'v2/contact'
-
-module Nylas
-  module V2
-  end
-end
```

Comment:
Don't need this anymore since the only stuff that will be in v1 and v2 are the only models that differ, AKA the Contact model.